### DISCUSSION TITLE
New IAG dynamic stall model

### DISCUSSION THREAD

**Discussion by aroseGWT** [OpenFAST/openfast#2288] [Ideas]
I'm wondering if anyone is planning on incorporating this dynamic stall model into OpenFAST/Aerodyn? Seems like it performs well for AoAs well beyond stall.
 
https://www.dnv.com/article/new-model-helps-engineers-predict-wind-turbine-standstill-vibration-in-extreme-conditions/

https://www.mdpi.com/1996-1073/16/10/3994

**Comment by jjonkman** [comment]
Dear @aroseGWT,

We are familiar with the new IAG unsteady airfoil aerodynamic model, but NREL is not currently funded to implement it within AeroDyn.  I agree that this would be a useful addition at some point (`UAMod` = 8?).

I'll move this Discussion into an Issue as a proposed feature enhancement.

Best regards,
